tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -692,7 +692,12 @@ def test_file_source_and_sink(self, security_protocol, exactly_once_source, conn
         metadata_quorum=[quorum.isolated_kraft],
         use_new_coordinator=[True, False]
     )
-    def test_bounce(self, clean, connect_protocol, metadata_quorum, use_new_coordinator=False):
+    @matrix(
+        metadata_quorum=[quorum.isolated_kraft],
+        use_new_coordinator=[True],
+        group_protocol=consumer_group.all_group_protocols
+    )
```

Comment:
We need the `clean` and `connect_protocol` values to be included.

```suggestion
    @matrix(
        clean=[True, False],
        connect_protocol=['sessioned', 'compatible', 'eager'],
        metadata_quorum=[quorum.isolated_kraft],
        use_new_coordinator=[True],
        group_protocol=consumer_group.all_group_protocols
    )
```